src/operations/References.cpp

```diff
@@ -59,10 +43,9 @@ std::vector<Reference> WorkspaceFolder::findAllReferences(Luau::TypeId ty, std::
     ty = Luau::follow(ty);
     auto ttv = Luau::get<Luau::TableType>(ty);
 
-    if (!ttv)
-        return {};
-
-    if (ttv->definitionModuleName.empty())
+    // When the definition module name is starts with '@', e.g. "@luau" or "@roblox", `LUAU_ASSERT(!buildQueueItems.empty());` in Frontend.cpp fails,
+    // after we call checkStrict below
+    if (!ttv || ttv->definitionModuleName.empty() || ttv->definitionModuleName[0] == '@')
```

Comment:
Ideally we map this to a definitions file in future.

Can you add a test and a TODO for this?